relayer/crates/starknet-chain-components/src/impls/queries/status_at_height.rs

```diff
@@ -0,0 +1,49 @@
+use cgp::prelude::*;
+use hermes_chain_components::traits::types::height::HasHeightType;
+use hermes_cosmos_chain_components::types::status::Time;
+use hermes_relayer_components::chain::traits::types::status::HasChainStatusType;
+use starknet::core::types::{BlockId, MaybePendingBlockWithTxHashes};
+use starknet::providers::{Provider, ProviderError};
+
+use crate::traits::provider::HasStarknetProvider;
+use crate::traits::queries::status_at_height::{
+    ChainStatusAtHeightQuerier, ChainStatusAtHeightQuerierComponent,
+};
+use crate::types::status::StarknetChainStatus;
+
+pub struct QueryStarknetChainStatusAtHeight;
+
+#[cgp_provider(ChainStatusAtHeightQuerierComponent)]
+impl<Chain> ChainStatusAtHeightQuerier<Chain> for QueryStarknetChainStatusAtHeight
```

Comment:
I think we can use [`CanQueryBlock`](https://github.com/informalsystems/hermes-sdk/blob/main/crates/chain/chain-components/src/traits/queries/block.rs#L11) instead. This is more of a block info. The chain status is used for knowing the current status of the chain.

Reply:
ac4eaf2